rocAL/include/pipeline/commons.h

```diff
@@ -247,6 +259,7 @@ enum class MelScaleFormula {
     SLANEY = 0,
     HTK
 };
+REGISTER_ENUM(MelScaleFormula)
```

Comment:
Non-blocking: Associated enum is defined as . Consider explicitly specifying the underlying type (e.g., ) to keep ABI stable and avoid surprises across compilers.